lib/ssl/src/tls_handshake_1_3.erl

```diff
@@ -1439,7 +1439,7 @@ process_certificate_request(#certificate_request_1_3{
                        maps:get(signature_algs, Extensions, undefined)),
     ServerSignAlgsCert = get_signature_scheme_list(
                            maps:get(signature_algs_cert, Extensions, undefined)),
-    CertAuths = get_certificate_authorites(maps:get(certificate_authorities, Extensions)),
+    CertAuths = get_certificate_authorites(maps:get(certificate_authorities, Extensions, undefined)),
```

Comment:
Here we need the default value of undefined, this is more difficult to test as you need a server that does not send the extension in its certificate request message. I will add this issue as an additional reason to extend our test framework.